ide/httpserver/src/org/netbeans/modules/httpserver/GrantAccessPanel.java

```diff
@@ -55,7 +55,7 @@ private void initComponents() {
         localTopMessage.setDisabledTextColor (javax.swing.UIManager.getColor ("Label.foreground"));  // NOI18N
         localTopMessage.setFont (javax.swing.UIManager.getFont ("Label.font")); // NOI18N
 
-        StringBuffer lTopMessage = new StringBuffer();
+        StringBuilder lTopMessage = new StringBuilder();
         lTopMessage.append(msg);
         localTopMessage.setText(lTopMessage.toString());
```

Comment:
Is lTopMessage even needed here? 
Wouldn't
 
localTopMessage.setText(msg != null ? msg : "null"); 
suffice ?